src/main/java/liquibase/ext/mongodb/configuration/MongoConfiguration.java

```diff
@@ -27,5 +29,11 @@ public class MongoConfiguration implements AutoloadedConfigurations {
                         "This will permit usage on Mongo Versions not supporting Validators")
                 .setDefaultValue(TRUE)
                 .build();
+
+        RETRY_WRITES = builder.define("retryWrites", Boolean.class)
+                .setDescription("Disabling this property will add retryWrites=false to connection URL" +
```

Comment:
You probably need a `. ` between these joined strings? Also maybe instead of saying "disabling this property..." something like "Setting this property to false..." might be more explicit and understandable?